### PR TITLE
Adds support for User-Agent string

### PR DESCRIPTION
**Pull request by rayalex**

### CODE REVIEW

File: allthingstalk/clients.py [hunk 1]
```diff
@@ -95,6 +96,7 @@ def prefix_http(url):
             self.mqtt = None
 
         self._devices = {}
+        self._version = self.__get_version()
```
Comment (vydd) [inline allthingstalk/clients.py]: thanks!

File: allthingstalk/clients.py [hunk 2]
```diff
@@ -200,8 +201,17 @@ def publish_asset_state(self, device_id, asset_name, state):
...
+                     headers=self._get_headers(),
+                     json = json_state)
+
+    def _get_headers(self):
+        return {
+            'Authorization': 'Bearer %s' % self.token,
+            'User-Agent': 'ATTalk-PythonSDK/%s' % self.__get_version()
+        }
+
+    def __get_version(self):
```
Comment (vydd) [inline allthingstalk/clients.py]: Why is the underscore here doubled?

File: allthingstalk/clients.py [hunk 3]
```diff
@@ -163,8 +164,8 @@ def create_asset(self, device_id, asset):
             'Profile': asset.profile
         }
         asset_dict = requests.post('%s/device/%s/assets' % (self.http, device_id),
-                                   headers={'Authorization': 'Bearer %s' % self.token},
-                                   json=attalk_asset).json()
+                                    headers=self._get_headers(),
```
Comment (vydd) [inline allthingstalk/clients.py]: weird spacing

**Review by vydd — CHANGES_REQUESTED** [review]
Just minor stuff; looks good otherwise!

**Comment by rayalex** [comment]
@vydd Can you re-check please?

**Review by vydd — APPROVED** [review]

**Comment by vydd** [comment]
Cool, have you tested it with a simple example? If so, feel free to merge and push the package to PyPI (or I can, if it's too much of a hassle).

**Comment by rayalex** [comment]
@vydd I have. I'll try out other examples and close this PR if everything passes.

**Comment by rayalex** [comment]
@vydd Works as expected.

**Pull request merged**